pkg/vpcmodel/nodesExplainability.go

```diff
@@ -213,31 +256,32 @@ func (c *VPCConfig) getRulesOfConnection(src, dst Node, conn *common.ConnectionS
 }
 
 // node is from getCidrExternalNodes, thus there is a node in VPCConfig that either equal to or contains it.
-func (c *VPCConfig) getContainingConfigNode(node Node) (Node, error) {
+func (c *VPCConfig) getContainingConfigNode(node Node) Node {
 	if node.IsInternal() { // node is not external - nothing to do
-		return node, nil
+		return node
 	}
 	nodeIPBlock := common.NewIPBlockFromCidr(node.Cidr())
-	if nodeIPBlock == nil { // string cidr does not represent a legal cidr
-		return nil, fmt.Errorf("could not find IP block of external node %v", node.Name())
+	if nodeIPBlock == nil { // string cidr does not represent a legal cidr, would be handled earlier
+		return nil
 	}
 	for _, configNode := range c.Nodes {
 		if configNode.IsInternal() {
 			continue
 		}
 		configNodeIPBlock := common.NewIPBlockFromCidr(configNode.Cidr())
 		if nodeIPBlock.ContainedIn(configNodeIPBlock) {
-			return configNode, nil
+			return configNode
 		}
 	}
-	return nil, fmt.Errorf("could not find containing config node for %v", node.Name())
+	return nil
```

Comment:
why is it valid to get in here? don't we expect any external node to be contained in a config node?

Reply:
Because a user may specify an IP that is internal but has to VSI connected to it, in which case we will get here. 
Obviously this means no connection exists.
We have an issue for this: once we finalize handling inputs perhaps we will not need it any more
I will add a ToDo